_assets/javascripts/components/pagination.js

```diff
@@ -156,8 +156,17 @@ $(document).ready(function() {
     navigateToPageByInput();
   }, false);
 
-  document.getElementById("pagination-current-page").addEventListener("keyup", function(e) {
-    if (e.key == 'Enter' && canNavigate) {
+  document.getElementById("pagination-current-page").addEventListener("keydown", function(e) {
+    if (e.keyCode == 189 || e.key == "-") {
```

Comment:
Could you set the numbers to a variable and then use that? Mainly so we do not have a bunch of magic numbers floating around? (See example below)

```
var something = 189;
var somethingElse = 95;
// ...
if (e.keyCode == something || e.key == "-") {
// ...
```

Reply:
On it!

Reply:
Code has been updated. Here is an updated netlify deploy preview:

 https://6148e137227ea71a0a66ac9c--crds-net.netlify.app/media/articles